LDK/resources/web/LDK/panel/TabbedReportPanel.js

```diff
@@ -764,19 +764,34 @@ Ext4.define('LDK.panel.TabbedReportPanel', {
             scope: this
         };
 
-        //special case these two properties because they are common
-        if (tab.report.viewName){
-            queryConfig.viewName = tab.report.viewName;
-        }
-
-        if (tab.report.containerPath){
-            queryConfig.containerPath = tab.report.containerPath;
-        }
-
-        //allow any other supported properties to be applied through here
-        if (tab.report.queryConfig){
```

Comment:
The idea of iterating the whitelist of configOptions is a good one, but are you sure no code uses report.queryConfig? It doesnt appear that's backward compatible

Reply:
Good point, i just did a text search in LabKey repo, and did not see usages for report.queryConfig. Also, will keep an eye on any test failures around this, if any.